sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -146,6 +146,12 @@ export class DataLakeFileClient extends DataLakePathClient {
     create(options?: FileCreateOptions): Promise<FileCreateResponse>;
     flush(position: number, options?: FileFlushOptions): Promise<PathUpdateResponse>;
     read(offset?: number, count?: number, options?: FileReadOptions): Promise<FileReadResponse>;
+    readToBuffer(buffer: Buffer, offset?: number, count?: number, options?: FileReadToBufferOptions): Promise<Buffer>;
```

Comment:
Do other languages also use `read`.  It feels nice to have `download` and `upload` pair.

Reply:
.NET is using `ReadTo`.